File: client/iih_stub/iih.ci_stub/iih.ci.ord/i/ICiTmplMainService.cs

using iih.ci.ord.dto.emsmain.tmpl;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace iih.ci.ord.i
{
    public interface ICiTmplMainService
    {
        /**
	 * 加载模板
	 * @param ems
	 * @return
	 */
        TmplRstDTO load(TmplLoadDTO ems);



        /**
         * 保存模板
         * @param ems
         * @return
         */
        TmplRstDTO save(TmplSaveDTO ems);
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/i/ems/ICiTmplMainService.java
package iih.ci.ord.i.ems;

import iih.ci.ord.d.ems.tmpl.TmplLoadDTO;
import iih.ci.ord.d.ems.tmpl.TmplRstDTO;
import iih.ci.ord.d.ems.tmpl.TmplSaveDTO;
import xap.mw.core.data.BizException;

public interface ICiTmplMainService {
	/**
	 * 加载模板
	 * @param ems
	 * @return
	 */
	public abstract TmplRstDTO load(TmplLoadDTO ems) throws BizException ;
	
	
	
	/**
	 * 保存模板
	 * @param ems
	 * @return
	 */
	public abstract TmplRstDTO save(TmplSaveDTO ems) throws BizException ;
}
